tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -43,6 +43,7 @@
 #include "symfun.h"
 #include "ilidir.h"
 #include "fdirect.h"
+#include <string>
```

Comment:
Based on [this build error](https://github.com/flang-compiler/flang/actions/runs/11842152761/job/33009497541?pr=1460), this `#include` statement should be placed above line 18 (`#include "machreg.h"`). In fact, all standard headers should be included before custom headers, including `stdlib.h`, `stdio.h`, etc.